server/crudtest.js

```diff
@@ -0,0 +1,60 @@
+const axios = require('axios');
```

Comment:
delete commented out code for readability?

Reply:
This is a small file that I created to test the endpoints -- I will add tests soon.